tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/ExportGroupPolicy.kt

```diff
@@ -48,19 +48,18 @@ class ExportGroupPolicy : Runnable, RestCommand() {
             }
         }
 
-        val groupPolicyFile = saveLocation?.let { File(it) } ?: getDefaultGroupPolicyFile()
-        groupPolicyFile.parentFile.mkdirs()
+        val groupPolicyFile = File(saveLocation)
+        groupPolicyFile.parentFile?.mkdirs()
+
+        if (!groupPolicyFile.exists()) {
+            groupPolicyFile.createNewFile()
+        }
+
```

Comment:
Can we do this if we make the change from my other comment?
```suggestion
        saveLocation.parentFile.mkdirs()
```